### ISSUE TITLE
How to tell CIBW to test wheels on more than one python version for each platform?

### ISSUE THREAD

**Issue by sdimovv** [pypa/cibuildwheel#1902]
### Description

I am building ABI3 (limited API) wheels with CIBW and I would to build the wheel only on the lowest Python version I support but tell CIBW to test it on all future Python versions for a given platform.

Concretely, I am targeting 3.7+. I want to build wheels for Python 3.7 only (for all major platforms) but test them on 3.7-3.12 (for each platform).

I've been thinking I can maybe use `nox` for this, but that still requires all of the desired Python interpreters be installed. 

Is there a way to achieve this with CIBW?

**Comment by henryiii** [comment]
Yes, it does this out of the box. If it builds an ABI3 wheel, each subsequent run with a higher Python version will simply use that wheel and test it.

**Comment by henryiii** [comment]
(As long as you do have a test-command set up, of course)
